Sources/Datadog/DatadogCore/DatadogCore.swift

```diff
@@ -283,17 +308,54 @@ extension DatadogCore: DatadogCoreProtocol {
         contextProvider.write { $0.featuresAttributes[feature] = attributes() }
     }
 
-    /* public */ func send(message: FeatureMessage, else fallback: @escaping () -> Void) {
+    /* public */ func send(message: FeatureMessage, sender: DatadogCoreProtocol, else fallback: @escaping () -> Void) {
         messageBusQueue.async {
             let receivers = self.messageBus.values.filter {
-                $0.receive(message: message, from: self)
+                $0.receive(message: message, from: sender)
             }
 
             if receivers.isEmpty {
                 fallback()
             }
         }
     }
+
+    /* public */ func flush() {
```

Comment:
leftover comment

Reply:
This was removed so it's not relevant anymore. But the `/* public */ func` was not a leftover - it is a convention we have behind annotating APIs that are `internal` today, but we will make them `public` in `2.0.0` release.